tekton-catalog/pipeline-loops/pkg/reconciler/pipelinelooprun/pipelinelooprun.go

```diff
@@ -229,10 +229,19 @@ func (c *Reconciler) reconcile(ctx context.Context, run *v1alpha1.Run, status *p
 	// Run is cancelled, just cancel all the running instance and return
 	if run.IsCancelled() {
 		if len(failedPrs) > 0 {
-			run.Status.MarkRunFailed(pipelineloopv1alpha1.PipelineLoopRunReasonCancelled.String(),
+			run.Status.MarkRunFailed(pipelineloopv1alpha1.PipelineLoopRunReasonFailed.String(),
+				"Run %s/%s was failed",
+				run.Namespace, run.Name)
+		} else {
+			reason := pipelineloopv1alpha1.PipelineLoopRunReasonCancelled.String()
+			if run.HasTimedOut() { // This check is only possible if we are on tekton 0.27.0 +
```

Comment:
I am trying to figure out, if we can somehow detect the tekton controller's version here. Otherwise, it may not work on 0.25.x versions.

Reply:
So, I have asked folks at tekton. It is not very straight forward at the moment.

We can probably remove this check until we move to 0.27 fully.

/cc @Tomcli

Reply:
Looks like this isn't a problem at all. To test it, I have installed a 0.25 version tekton controller and tried to cancel a pipelineRun with code in this PR.